docs/bonsai-ui/checkout/events.mdx

```diff
@@ -0,0 +1,170 @@
+| `shipping_info_entered`        | Shipping information has been successfuly entered, and the user has clicked "Continue to payment" button |
+| `payment_info_entered`         | Payment information has been successfuly entered, and the user has clicked "Submit order" button         |
+| `order_completed`              | Order has been completed, and the user was redirected to Order Success page                              |
+| `product_unavailable`          | One or more products are unavailable                                                                     |
+| `variant_does_not_exist`       | One or more variants do not exist                                                                        |
+| `product_does_not_exist`       | One or more products do not exist                                                                        |
+| `products_price_change`        | One or more products have changed price                                                                  |
+| `international_shipping_error` | International shipping is not available for one or more products                                         |
+| `invalid_input`                | Invalid input has been provided                                                                          |
+| `tax_error`                    | An error has occured during tax calculation                                                              |
+| `taxes_or_duties_error`        | An error has occured during taxes or duties calculation                                                  |
+| `province_unrecognized`        | Province is not recognized                                                                               |
+| `province_not_found`           | Province is not found                                                                                    |
+| `payment_method_not_supported` | Payment method is not supported                                                                          |
+| `payment_required`             | Payment is required                                                                                      |
+| `out_of_stock`                 | One or more products are out of stock                                                                    |
+| `insufficient_inventory`       | One or more products have insufficient inventory                                                         |
+| `other_error`                  | An error has occured during the checkout process                                                         |
+| `express_checkout_mounted`     | Express Checkout button has been rendered                                                                |
+| `express_checkout_begin`       | User has clicked on the Express Checkout button                                                          |
```

Comment:
Just this event might be better to change the name so it is consistent with the other past tense of the non-error events, maybe `express_checkout_begun`